model/user.py

```diff
@@ -11,12 +11,12 @@ class User(db.Model):
     user_type = db.Column(db.Enum(UserType), default=0)
 
     username = db.Column(db.String(128), unique=True, nullable=False)
-    full_name = db.Column(db.String())
-    mail = db.Column(db.String())
-    phone = db.Column(db.String())
-    affiliation = db.Column(db.String())
+    full_name = db.Column(db.String(128))
+    mail = db.Column(db.String(128))
+    phone = db.Column(db.String(128))
+    affiliation = db.Column(db.String(128))
 
-    cached_password = db.Column(db.String())
+    cached_password = db.Column(db.String(128))
```

Comment:
I don't think we need to enforce character limits on the columns, should we simply remove them?
If we don't remove them, 128 seems a bit low for full names and cached passwords (we have students with REALLY long names after all.)